### DISCUSSION TITLE
Per-user log files?

### DISCUSSION THREAD

**Discussion by nrvale0** [regolith-linux/regolith-desktop#902] [Q&A]
I promise I went through the docs -- and even asked various LLVMs ;) -- but I've not found an answer to the question of where regolith might be dumping per-user log files? ~/.xessession-error* is what I sort of expected might be the right place to look but it sure doesn't look like that is correct.

More background: 

I do a lot of bouncing about between 2.2 and 3.0 on Debian and the Ubuntus. I'm on the i3 side right now but would like to test sway for some scaling and zooming I would like to implement. Inevitably I'm running into a lot of problems / differences in the software matrix due to all of the jumping around. Currently I have created a fresh uid just for testing 3.0 with sway. But when I try to login to that account with sway session it immediately exits back to the greeter. Same account with i3 works just fine. So that's why I'm looking for per-user log files. 

thx

**Comment by kgilmer** [comment]
Nothing comes to mind immediately regarding session logs, but in these "display manager" issues, I find that opening a TTY (eg: `ctrl alt F3` on a US keyboard) and tailing `/var/log/syslog` (or whichever log you're after) works well.

**Comment by nrvale0** [comment]
Y, systemd sucks up all of the logs and shoves into various places, not /var/log/{messages,syslog} where they are hard to find. As is its SOP.

I'm going to set up a log aggregation host and see if I can't correlate weirdness across the set of hosts. 

If there's any way you are aware of to tweak display manager to dump stuff to ~/.xsession-error* or similar it'd be a welcome tweak to the environment, IMO.
